draft-ietf-jsonpath-base.md

```diff
@@ -168,13 +168,19 @@ Value:
   composed of components such as containers, namely JSON objects and arrays, and
   atomic data, namely null, true, false, numbers, and text strings.
 
-Member:
-: A name/value pair in an object.  (Not itself a value.)
+String:
+: A sequence of UTF-8 characters surrounded by either single quotes (`)
+  or double quotes (").
+  Escaping quote characters is required only for the specific character
+  which surrounds the string.
+  For example, a double quote character (") within a string surrounded
+  by single quotes (') does not need to be escaped.
+  Single quotes are preferred.
 
 Name:
 : The name in a name/value pair constituting a member.  (Also known as
   "key", "tag", or "label".)
-  This is also used in {{-json}}, but that specification does not
+  This is also used in {{-json}}, but that specification does not 
```

Comment:
SO do you want to keep the deletion of "member" and this and the below changes (which would unfix some nits)?